float64 VizJointLen
float64 VizJointRad
float64 FSThick
float64 FSSize
float64 BaseLenX
float64 BaseLenY
float64 BaseLenZ
float64 LegRad
float64 LegLen1
float64 LegLen2
float64 EyeCylR
float64 EyeCylL
float64 FootRad
float64 EyeRad
float64 BaseDensity
float64 OtherDensity

int32 TerrainMode

int32 MaxContacts
float64 TimeStep
float64 Gravity
bool EnableKeyEvent

float64 HingeFMax
int32 ControlMode

